src/Scene/NodeProcess.js

```diff
@@ -251,29 +245,37 @@ function updateNodeImagery(scene, quadtree, node, layersConfig, force) {
             }
         }
 
-        const args = {
-            layer,
-        };
+        let ancestor = null;
 
         const currentLevel = node.materials[RendererConstant.FINAL].getColorLayerLevelById(layer.id);
         // if this tile has no texture (level == -1), try use one from an ancestor
         if (currentLevel === -1) {
-            args.ancestor = findAncestorWithValidTextureForLayer(node, l_COLOR, layer);
+            ancestor = findAncestorWithValidTextureForLayer(node, l_COLOR, layer);
         } else {
             var targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node.level, currentLevel, layer.updateStrategy.options);
             if (targetLevel === currentLevel) {
                 continue;
             }
             if (targetLevel < node.level) {
-                args.ancestor = node.getNodeAtLevel(targetLevel);
+                ancestor = node.getNodeAtLevel(targetLevel);
             }
         }
 
         node.layerUpdateState[layer.id].newTry();
 
-        promises.push(quadtree.interCommand.request(args, node, refinementCommandCancellationFn).then(
+        const command = {
+            /* mandatory */
+            layer,
+            requester: node,
+            priority: nodeCommandQueuePriorityFunction(node),
+            earlyDropFunction: refinementCommandCancellationFn,
+            /* specific params */
+            ancestor,
+        };
+
+        promises.push(quadtree.scheduler.execute(command).then(
```

Comment:
There's more interest in keeping the scheduler in the Quadtree, as interfaceCommand is removed?